docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md

```diff
@@ -1,363 +1,628 @@
+**B.** A front-end **User** is someone who interacts with your project through the front-end. A front-end **User** can, for example, be an "Author" of an article, make a purchase, has an account, leaves a review, or leaves a comment.
 
-Another example, could be a **Content Type** called `menu` which is made up of `menu items`. These could have a `name`, `description of the menu item`, an `image` and perhaps even a `list of ingredients`.
+Up until this point, you have created an **Administrator**, and so you next want to create a front-end **User**.
+
+**Note:** It is not necessary to always create a front-end **User** for your **Administrators**; in this case, the **Administrator** is also a front-end **User** as an "Author" of content in the application.
+
+- Click on `Users` located under **CONTENT TYPES** in the left-hand menu
+- Click the blue **+ Add New User** button in the top right corner
+- Next, complete the `Username`, `Email`, and `Password` fields
+- Select `ON` for the **Confirmed** toggle field
+- To the right, under **Role**, select `Authenticated`
+- Save the new user by clicking the blue **Save** button (top right)
+
+![New front-end User](../assets/getting-started/tutorial/new-front-end-user.png 'New front-end User')
+
+You are now ready to create your first **Content Type**.
+
+## 3. Create a new Content Type called "Restaurant"
+
+**Content Types** are a collection of entered data represented by fields. For example, a **Content Type** called `Restaurant` may be intended to display information regarding restaurants. A `restaurant` **Content Type** could have fields that include a `name`, the main `image`, and a `description` - _at a minimum_. However, a `restaurant` could also have a `category` or multiple `categories`, and a `restaurant` could perhaps need to show `hoursofoperation`.
+
+A **Content Type** can be considered a sort of _blueprint_ for the data created. In other words, a **Content Type** is the schema of the data structure.
+
+The next section guides you through the steps needed for each of these above **Content Type** fields.
 
 ::: tip NOTE
 
-More **Restaurant** themed **Content Types** can be seen in the Strapi demo site: [Strapi Foodadvisor](https://foodadvisor.strapi.io/).
+Additional **Restaurant** themed **Content Types** and fields can be seen in the [FoodAdvisor demo site](https://foodadvisor.strapi.io/).
 
 :::
 
-At this point, your project is empty. You need to create a new **Content Type**.
+### The Restaurant Content Type
+
+- Go to the **Content Type Builder** plugin, located in the left menu: Under **PLUGINS**: --> **Content Type Builder**. You are now able to see the three available **Content Types**.
```

Comment:
You never use `.` at the end of all list lines. Expect here.

Reply:
These have all been fixed now.